common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -573,12 +573,10 @@ def source_library_values(self):
         """
         lib_tools = self.runtime.service(self, 'library_tools')
         user_perms = self.runtime.service(self, 'studio_user_permissions')
-        all_libraries = lib_tools.list_available_libraries()
-        if user_perms:
-            all_libraries = [
-                (key, name) for key, name in all_libraries
-                if user_perms.can_read(key) or self.source_library_id == unicode(key)
-            ]
+        all_libraries = [
```

Comment:
Why was the check for a `None` `user_perms` removed here? Is it guaranteed to be non-`None`?